src/components/fieldset/template.test.js

```diff
@@ -61,6 +61,15 @@ describe('fieldset', () => {
     expect($legend.text().trim()).toEqual('What is your address?')
   })
 
+  it('allows you to set the initial aria-describedby attribute', () => {
```

Comment:
'Initial' here suggests that `describedBy` may still change, but it's outputted as it is.

```suggestion
  it('allows you to set the aria-describedby attribute', () => {
```